http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/common/Constants.java

```diff
@@ -134,7 +134,9 @@
 
     public static final String PROTOCOL = "PROTOCOL";
 
-    public static final String PROTOCOL_NAME = "http";
+    public static final String SCHEME_HTTP = "http";
+
+    public static final String SCHEME_HTTPS = "https";
```

Comment:
On second thoughts, would it sound better if we say HTTP_SCHEME instead of SCHEME_HTTP ?